inttest/common/footloosesuite.go

```diff
@@ -1444,3 +1445,37 @@ func (s *FootlooseSuite) AssertSomeKubeSystemPods(client *kubernetes.Clientset)
 
 	return false
 }
+
+type IPV6Config struct {
+	IPv6        bool   `json:"ipv6"`
+	FixedCidrV6 string `json:"fixed-cidr-v6"`
+}
+
+func (s *FootlooseSuite) ConfigureIPv6Docker(enabled bool) error {
```

Comment:
This is something that's host-specific. Better put it into the GitHub workflows? Maybe as a shell script that's invoked as part of a `run` script?

Reply:
That sounds as a good idea i can look into it.
restarting the docker daemon must also be done before the footloose suite creates all containers.

Reply:
One way to test if IPv6 is enabled could be to check if the default docker bridge network has EnableIPv6 set to true.

Reply:
Should we bail out with an error or not ?